ibm/resource_ibm_cis_healthcheck.go

```diff
@@ -2,331 +2,364 @@ package ibm
 
 		Schema: map[string]*schema.Schema{
-			"cis_id": {
+			cisID: {
 				Type:        schema.TypeString,
 				Description: "CIS instance crn",
 				Required:    true,
 			},
-			"path": {
+			cisGLBHealthCheckID: {
 				Type:        schema.TypeString,
-				Description: "path",
-				Optional:    true,
+				Computed:    true,
+				Description: "GLB Monitor/Health check id",
+			},
+			cisGLBHealthCheckPath: {
+				Type:         schema.TypeString,
+				Description:  "path",
+				Optional:     true,
+				ValidateFunc: validateURLPath,
 			},
-			"expected_body": {
+			cisGLBHealthCheckExpectedBody: {
 				Type:        schema.TypeString,
 				Description: "expected_body",
 				Optional:    true,
 			},
-			"expected_codes": {
+			cisGLBHealthCheckExpectedCodes: {
 				Type:        schema.TypeString,
 				Description: "expected_codes",
 				Optional:    true,
 			},
-			"description": {
+			cisGLBHealthCheckDesc: {
 				Type:        schema.TypeString,
 				Description: "description",
 				Optional:    true,
 			},
-			"type": {
-				Type:         schema.TypeString,
-				Description:  "type",
-				Optional:     true,
-				Default:      "http",
-				ValidateFunc: validateAllowedStringValue([]string{"http", "https", "tcp"}),
-			},
-			"method": {
-				Type:         schema.TypeString,
-				Description:  "method",
-				Optional:     true,
-				ValidateFunc: validateAllowedStringValue([]string{"GET", "HEAD"}),
-			},
-			"timeout": {
-				Type:        schema.TypeInt,
-				Description: "timeout",
+			cisGLBHealthCheckType: {
+				Type:        schema.TypeString,
+				Description: "type",
 				Optional:    true,
-				Default:     5,
-				//ValidateFunc: validation.IntBetween(1, 10),
+				Default:     "http",
+				ValidateFunc: validateAllowedStringValue(
+					[]string{
+						"http",
+						"https",
+						"tcp",
+					},
+				),
 			},
-			"retries": {
-				Type:        schema.TypeInt,
-				Description: "retries",
+			cisGLBHealthCheckMethod: {
+				Type:        schema.TypeString,
+				Description: "method",
 				Optional:    true,
-				Default:     2,
-				//ValidateFunc: validation.IntBetween(1, 5),
+				Default:     "GET",
+				ValidateFunc: validateAllowedStringValue(
+					[]string{
+						"GET",
+						"HEAD",
+					},
+				),
 			},
-			"interval": {
-				Type:        schema.TypeInt,
-				Description: "interval",
-				Optional:    true,
-				Default:     60,
+			cisGLBHealthCheckTimeout: {
+				Type:         schema.TypeInt,
+				Description:  "timeout",
+				Optional:     true,
+				Default:      2,
+				ValidateFunc: validatePortRange(1, 10),
+			},
+			cisGLBHealthCheckRetries: {
+				Type:         schema.TypeInt,
+				Description:  "retries",
+				Optional:     true,
+				Default:      1,
+				ValidateFunc: validatePortRange(1, 3),
+			},
+			cisGLBHealthCheckInterval: {
+				Type:         schema.TypeInt,
+				Description:  "interval",
+				Optional:     true,
+				Default:      60,
+				ValidateFunc: validatePortRange(5, 3600),
```

Comment:
Please use the latest way of validation
you can refer ratelimit resource [here](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/ibm/resource_ibm_cis_rate_limit.go#L244)

Reply:
added resourceIBMCISHealthCheckValidator() method